roles/etcd/meta/main.yml

```diff
@@ -18,3 +18,4 @@ galaxy_info:
 dependencies:
 - { role: os_firewall }
 - { role: etcd_common }
+- { role: etcd_server_certificates }
```

Comment:
Why break up the etcd_certificates role here? instead you can just call etcd_certificates with a variable for generating either a client cert or a server cert.

Also, now that I think about it with the delegate_to, we could probably also consolidate the etcd_ca and etcd_certificates role (I would suggest they be separate task files within the etcd role and included in main.yml). That consolidation may make it harder to generalize the CA and integrate it with an external cert provider eventually though.